CHANGES.md

```diff
@@ -1,3 +1,30 @@
+## edge-19.12.2
+
+* CLI
+  * Added support for injecting CronJobs and ReplicaSets, and the ability to use
+    them as targets in the CLI subcomands
+  * Added ability to update the trust anchor and issuer certificates during
+    `linkerd upgrade` with the new flags `--identity-issuer-certificate-file`,
+    `--identity-issuer-key-file` and `identity-trust-anchors-file`
+
+* Controller
+  * Fixed bug causing pods having security context capabilities to fail to be
+    injected
+  * Upgraded Prometheus to v1.2.1 (thanks @daxmc99!)
+
+* Web UI
+  * Added support for CronJobs and ReplicaSets, including new Grafana dashboards
+    for them
+
+* Proxy
+  * Increased destination controller client's connection receive window to avoid
+    it becoming exhausted, by single stalled streams
+  * Eagerly read profiles off the wire
```

Comment:
These are sort of internals-focused -- the user facing impact here is that we believe we've addressed a long-standing service discovery staleness bug.